### PR TITLE
rework egress proxy

### PR DESCRIPTION
**Pull request by robert-bryson**
As there is more flexibility with passing inputs in Github actions (thanks for the find, @nickumia-reisys!), the egress proxy enable/disable can be simplified into one action.

### CODE REVIEW

File: .github/workflows/toggle-egress.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      options:
+        - development
+        - staging
+        - prod
+
+jobs:
+  toggle-egress:
+    concurrency: ${{ github.event.inputs.environ }}
+    name: ${{github.envent.inputs.toggle-egress}} egress (${{ github.event.inputs.environ }})
+    environment: ${{ github.event.inputs.environ }}
```
Comment (nickumia-reisys) [inline .github/workflows/toggle-egress.yml]: Did Tyler or Jin already change the Github Action Environments?

**Review by nickumia-reisys — DISMISSED** [review]
💯

File: .github/workflows/toggle-egress.yml (hunk 1 above)
Comment (Jin-Sun-tts) [reply .github/workflows/toggle-egress.yml]: I did not change the 'production' to 'prod' in data.gov

**Comment by robert-bryson** [comment]
Related to https://github.com/GSA/data.gov/issues/3996

**Review by Santanathegreat — APPROVED** [review]

**Review by fepimentel — APPROVED** [review]

**Review by Santanathegreat — APPROVED** [review]

**Review by fepimentel — APPROVED** [review]

**Comment by btylerburton** [comment]
can we close this?

**Comment by robert-bryson** [comment]
> can we close this?

I think yeah, probably. The intention is still valid, I think, but this PR can probably be closed.

**Review by fepimentel — COMMENTED** [review]
Improvements

**Comment by robert-bryson** [comment]
> Improvements

I'll never understand this spam, but it does appear that the bots are using generated profile pictures now. I'm not sure if that makes it better or worse.

**Pull request closed without merging**